apps/labs/labs-mokainosspringboot/labs-mokainosspringboot.yaml

```
@@ -7,9 +7,13 @@ metadata:
 spec:
   releaseName: labs-mokainosspringboot
   chart:
-    git: git@github.com:hmcts/labs-mokainosspringboot
-    ref: master
-    path: charts/labs-mokainosspringboot
+    spec:
+      chart: ./charts/labs-mokainosspringboot
+      sourceRef:
+        kind: GitRepository
+        name: labs-mokainosspringboot
+        namespace: labs
```

should be this afaik
```suggestion
      sourceRef:
        kind: GitRepository
        name: hmcts-charts
        namespace: flux-system
```